src/praveenReddyDec2019/assignment2/Quetion3.java

```diff
@@ -0,0 +1,31 @@
+package praveenReddyDec2019.assignment2;
+
+//Check if array contains a number in Java?
+
+public class Quetion3 {
+	public void ArrayContains()
+	{
+		int a[]  = {2,5,9,7,4,6};
+		 int b = a.length;
+		 boolean found = false;
+		for(int i=1;i<b;i++)	{
+			if(a[i]==6)	{
```

Comment:
6 is hard code value we have to write in method parameter, what is user wanted to use your method.